tests/helpers.bash

```diff
@@ -26,7 +26,7 @@ function teardown() {
 function createrandom() {
 	dd if=/dev/urandom bs=1 count=${2:-256} of=${1:-${BATS_TMPDIR}/randomfile} status=none
 	# Set the mtime to the epoch so it won't be different once it is deduplicated with OSTree
-	touch -t 7001010000.00 ${1:-${BATS_TMPDIR}/randomfile}
+	touch -d 1970-01-01T00:00:00Z ${1:-${BATS_TMPDIR}/randomfile}
```

Comment:
Given the huge grain-of-salt in which I'm completely clueless about this, with no understanding of ostree nor the tests that require this... might it not be a lot clearer and friendlier to just say `-d @0`?

I'm not going to gate on it, because I don't know this code base at all. If you agree, though, and re-push, could I ask you to refactor the evil duplication, by maybe defining `randomfile=${1...etcetcetc` ?

Reply:
Sadly, the FreeBSD touch command doesn't support the @<integer> format for time:
```
$ touch -d @0 foo                                                                                                                                                           1 ↵
touch: out of range or illegal time specification: YYYY-MM-DDThh:mm:SS[.frac][tz]
```

The format I chose is the only one I can see which is supported on both FreeBSD and Linux. I can add a comment to clarify. I also saw the evil copypasta - I'll fix that.

Reply:
Ah - thanks for the explanation. You must have your hands full, compensating for gnu-toolchain assumptions in our code base.

Reply:
Its not too bad so far :)